docs/plugins/best_practices.md

```diff
@@ -229,3 +229,116 @@ up to report test coverage, but you can test locally as well, using
    from coverage with the comment `# pragma: no cover`
 5. In the cookiecutter, coverage tests from github actions will be uploaded to
    codecov.io
+
+## Set style for additional windows in your plugin
+
+In napari plugins we strongly advise to use widget docked in napari window,
+but sometimes is required to use separate window. 
+The best practice is to use [`QDialog`](https://doc.qt.io/qt-5/qdialog.html)
+based windows with parent set to widget
+already docked in napari window (sample code snip on the end).
+
+```python
+from qtpy.QtWidgets import QDialog, QWidget, QSpinBox, QPushButton, QGridLayout, QLabel
+
+class MyInputDialog(QDialog):
+    def __init__(self, parent: QWidget):
+        super().__init__(parent)
+        self.setWindowTitle("My Input Dialog")
+        self.number = QSpinBox()
+        self.ok_btn = QPushButton("OK")
+        self.cancel_btn = QPushButton("Cancel")
+        
+        layout = QGridLayout()
+        layout.addWidget(QLabel("Number:"), 0, 0)
+        layout.addWidget(self.number, 0, 1)
+        layout.addWidget(self.ok_btn, 1, 0)
+        layout.addWidget(self.cancel_btn, 1, 1)
+        self.setLayout(layout)
+        
+        self.ok_btn.clicked.connect(self.accept)
+        self.cancel_btn.clicked.connect(self.reject)
+        
+class MyWidget(QWidget):
+  def __init__(self, viewer: "napari.Viewer"):
+        super().__init__()
+        self.viewer = viewer
+        self.open_dialog = QPushButton("Open dialog")
+        self.open_dialog.clicked.connect(self.open_dialog_clicked)
+        
+    def open_dialog(self):
+        dialog = MyInputDialog(self)  
+        # this setting parent to self allow to inheritance of style from napari window
```

Comment:
```suggestion
        # setting parent to self allows the dialog to inherit its style from the viewer
```

I would probably move the comment above the line assigning dialog, so it's clear which line we're referring to (unless it's actually on the same line in which case ignore me sorry)